v3/plugins/che-incubator/theia-dev/0.0.1/meta.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
```

Comment:
@garagatyi looks a bit misleading that `v3` plugins have `v2` API version. WDYT?

Reply:
answered below